CHARTER.md

```diff
@@ -0,0 +1,71 @@
+
+TF is leveraging the existing editing [GitHub](https://github.com/w3c/editing) repo with well-known locations and history of discussions on a variety of editing-related topics in order to “incubate” new editing-specific standards.
+
+This charter is intended to reflect on the current direction of the TF group, so that there is common agreement. It may be altered at any point in order to reflect new priorities or work items. 
+  
+
+### 1. Scope
+
+The scope of Editing TF covers multiple aspects of editing, which may  include:
+
+- Textual input and text manipulation 
+- Text editing related events 
+- Selection 
+- Clipboard 
+- Spellcheck and grammar checking 
+- Reusing or discarding parts of or entire pre-existing editing APIs to serve as primitives of editing systems 
+- Standardization of under-specified existing editing related features 
+- Highlighting parts of text ranges without inducing DOM mutations. 
+
+Experts who are contributing members of the task force will contribute to documents produced by the group, as described in the section on deliverables below. 
+ 
+
+### 2. Deliverables
+
+This Task Force will
+- ensure that browsers are developing APIs that meet the expectations of editing framework authors of varying skill and team sizes.   
+- allow a network of experts to share information about gaps and requirements for building editing frameworks on the web.  
+
+The specifications the Editing Taskforce is producing may be driven to the spec level directly by its members within the Web Applications Working Group, or be transferred to another, more appropriate working group after initial collection of requirements. Furthermore, the group defines its success as when it at the very least, produces a recommendation with well defined requirements that result in enhancing existing text editing related specification drafts or helps design new ones.
+
+The TF is expected to work on the following efforts until the end of the charter but it is not meant to be an exhaustive list:
+
+- [Async Clipboard API](https://bugs.chromium.org/p/chromium/issues/detail?id=931839) 
+- [ContentEditableDisabled](http://w3c.github.io/editing/contentEditableDisabled.html) - ability to disable system UI 
+- [EditContext API](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/EditContext/explainer.md) 
+- [Highlight API](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/highlight/explainer.md) 
+- Native Selection and Caret behaviors 
+- [SpellChecker API](https://github.com/w3c/editing/issues/166) 
+- [SIP Policy](https://github.com/whatwg/html/issues/4876) 
+- [Input Events](https://www.w3.org/TR/input-events-1/) 
+- [ContentEditable](https://w3c.github.io/contentEditable/) 
+
```

Comment:
@ylafon + @siusin, some of these are not in our charter, and given that "Task forces do not publish technical reports", how can we include these? Do we need to recharter the Web Apps WG?

Reply:
If anything there is on TR and not in WebApps's charter, I guess that a rechartering (or at least a discussion about it) is appropriate. For those that are just ED's with no presence on TR, I'd consider these under incubation, which is an informal affair that can happen anywhere. They could be added to the charter now, but I don't think it's particularly urgent to do so until we try to publishing them as FPWD (if we're there already, then yes, they should be added).

Reply:
> For those that are just ED's with no presence on TR, I'd consider these under incubation, which is an informal affair that can happen anywhere. 

Generally speaking, the preference is that incubations happen at the WICG - but it would be ok to incubate things in the WebApps WG as long as they appropriately marked as incubations (i.e., they be clearly labelled "unofficial" until such time that they are not). I sympathetic that everyone who needs to be part of the discussion is following the w3c/editing repo and associated mailing list, which is why I'm also supportive of incubating within WebApps. 

> They could be added to the charter now, but I don't think it's particularly urgent to do so until we try to publishing them as FPWD (if we're there already, then yes, they should be added).

Yes, that makes sense.

Reply:
AFAIK, respec takes care of marking a spec as not being official as long as one marks it as an editor's draft and not a working draft beyond FPWD. That's really the distinction there is right? Either it's pre-FPWD in which case it's "unofficial" or it is post-FPWD in which case it starts being recognized by the W3C and then psot-rec when it really is soemthing with the W3C's stamp of approval on. 

Or do we need to make a distinction between official and unoffical pre-FPWD editor's draft?

Reply:
All we need to do is set `"specStatus": "unofficial"` in the ReSpec config. Same thing we did with `execCommand`.